src/models/thread.ts

```diff
@@ -207,16 +211,17 @@ export class Thread extends ReadReceipt<EmittedEvents, EventHandlerMap> {
         if (!toStartOfTimeline) {
             room!.addLocalEchoReceipt(event.getSender()!, event, ReceiptType.Read);
         }
-        this.onEcho(event);
+        this.onNewReply(event, !toStartOfTimeline);
     };
 
-    private onEcho = async (event: MatrixEvent): Promise<void> => {
+    private onNewReply = async (event: MatrixEvent, emit: boolean): Promise<void> => {
         if (event.threadRootId !== this.id) return; // ignore echoes for other timelines
         if (this.lastEvent === event) return;
-        if (!event.isRelation(THREAD_RELATION_TYPE.name)) return;
```

Comment:
I am struggling to see why this go removed

Reply:
Edits don't have a thread relation, but should trigger downstream events all the same.